contributor/Member_Application.md

```diff
@@ -0,0 +1,8 @@
+# Member Application
+To be a member of OpenLab follow the simple steps below:
+
+1. Open an issue to [apply to join](https://github.com/theopenlab/openlab/issues/new?template=joinin.md) first, fill out the form completely with as much detail, for example your contact information and the way you will support to OpenLab
+2. The governance team owner will assign a role to you based on your information, then will contact you and send you an invitation email, please confirm it and reply to us
```

Comment:
The OpenLab governance team owner will join you into [team](https://github.com/orgs/theopenlab/teams) based on your information, then will contact you and send you an invitation in github.com, please pay attention on Github [notifications](https://github.com/notifications) and confirm it.